doc/releases.md

```diff
@@ -1,5 +1,115 @@
 # Releases
 
+## Version 1.19
+
+### Version 1.19.0
+
+**June 13, 2024**
+
+This release includes an exciting new feature popups, this will make it easier to dive deeper and interact with your data.
```

Comment:
Most exciting and awaited feature (10yrs) is of course support for numerical/ timeseries access for Bars plots 😄